support-frontend/conf/routes

```diff
@@ -109,7 +109,7 @@ GET  /uk/subscribe/paper/delivery                                  controllers.P
 GET  /subscribe/paper/checkout                                     controllers.PaperSubscription.displayForm()
 
 # redemptions
-GET  /subscribe/redeem/welcome/:redemptionCode                     controllers.RedemptionController.displayForm(redemptionCode: String)
+GET  /subscribe/redeem                                             controllers.RedemptionController.displayForm(redemptionCode: String)
```

Comment:
good idea, I hit this this morning when I was trying to demo a blank redemption page.  I assume the /r/<code> redirect will still work?

Reply:
Yes it does, it converts the code to a querystring